stylesheets/_component.loading.scss

```diff
@@ -96,6 +96,12 @@
         border: 0;
         text-indent: 0;
     }
+
+    .lt-ie10 & {
+        margin: 0 0 -2px;
+        background: url('../images/ie-loading.gif') center center no-repeat;
```

Comment:
Good improvement, but could you use https://github.com/jadu/pulsar/blob/develop/images/loading.gif instead to match the non-gif one?